docs/tutorial.rst

```diff
@@ -198,6 +198,8 @@ Hy.  Let's experiment with this in the hy interpreter::
   {'dog': 'bark', 'cat': 'meow'}
   => (, 1 2 3)
   (1, 2, 3)
+  => #{3 1 2}
+  set([1L, 2L, 3L])
```

Comment:
I think this looks better on versions of Python released during the last half-decade&mdash;
```
=> #{3 1 2}
{1, 2, 3}
```
(If I didn't have a strict personal policy of never using smiley emoticons, I would insert one here to indicate that this is said in a tone of good-natured humor rather than actual antipathy towards Python 2.)